app/connectors/CustomsFinancialsApiConnector.scala

```diff
@@ -160,6 +168,71 @@ class CustomsFinancialsApiConnector @Inject()(httpClient: HttpClientV2,
       logger.error(s"Unable to download CSV :${e.getMessage}")
       Left(UnknownException)
   }
+
+  def postCashAccountStatementRequest(eori: String,
+                                      can: String,
+                                      from: LocalDate,
+                                      to: LocalDate)
+                                     (implicit hc: HeaderCarrier): Future[Either[ErrorResponse, AccountResponseCommon]] = {
+
+    val request = CashAccountStatementRequestDetail(eori, can, from.toString, to.toString)
+
+    httpClient.post(url"$retrieveCashAccountStatementsUrl")
+      .withBody[CashAccountStatementRequestDetail](request)
+      .execute[AccountResponseCommon]
+      .map(processStatusCode)
+
+  }.recover {
+    case UpstreamErrorResponse(_, BAD_REQUEST, _, _) =>
+      logger.error("BAD Request for postCashAccountStatements")
+      Left(BadRequest)
+
+    case UpstreamErrorResponse(_, INTERNAL_SERVER_ERROR, _, _) =>
+      logger.error("No Transactions available for postCashAccountStatements")
+      Left(NoTransactionsAvailable)
+
+    case UpstreamErrorResponse(_, SERVICE_UNAVAILABLE, _, _) =>
+      logger.error("SERVICE_UNAVAILABLE for postCashAccountStatements")
+      Left(NoTransactionsAvailable)
+
+    case e =>
+      logger.error(s"Unknown error for postCashAccountStatements :${e.getMessage}")
+      Left(UnknownException)
+  }
+
+  private def processStatusCode(response: AccountResponseCommon): Either[ErrorResponse, AccountResponseCommon] = {
+
+    response.statusText match {
+
+      case Some(helpers.Constants.REQUEST_COULD_NOT_BE_PROCESSED) =>
+        logger.error(s"REQUEST_COULD_NOT_BE_PROCESSED for the postCashAccountStatementRequest - processStatusCode")
+        Left(RequestCouldNotBeProcessed)
+
+      case Some(helpers.Constants.DUPLICATE_SUBMISSION) =>
```

Comment:
Also, import DUPLICATE_SUBMISSION and other values and just use as below (no need to mention helpers.Constants)

 case Some(DUPLICATE_SUBMISSION)